### PR TITLE
Creates data genScript

### PR DESCRIPTION
**Pull request by danzerca**
Starts load script. Work to still be done.

@PeanutJack please review my seed.js file. I started my load.js file, but it cannot yet handle the large text file.

### CODE REVIEW

File: cqlDB/seed.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+const fs = require('fs');
+const faker = require('faker');
+
+// use mongodb for generating script
+const db = require('./index.js');
+const Product = require('./model.js');
```
Comment (PeanutJack) [inline cqlDB/seed.js:6]: If you're splitting the load script into another file, I don't think you need to import the db here.

File: cqlDB/seed.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    console.log('Time to write (ms): ', Date.now() - start);
+    console.log(idCount, i);
+  }
+}
+
+
+write();
+
+module.exports = write;
+module.exports = generateEntry;
```
Comment (PeanutJack) [inline cqlDB/seed.js:89]: module.exports is by default an empty object. If you want to export multiple things you can attach them to keys on the object. Here you're replacing the whole object twice.

File: cqlDB/seed.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  for (var i = 0; i < numOfVariations; i++) {
+    var randomColor = faker.commerce.color(); // Fender Stratocaster
+    var randomCost = faker.commerce.price(); // 492.00
+    var numOfImages = Math.ceil((Math.random() * 5));
+    var randomImages = [] // 4 random image urls
+
+    var createImage = function() {
+      var url = 'http://picsum.photos/seed/'
+      var urlend='/846/1038'
```
Comment (PeanutJack) [inline cqlDB/seed.js:29]: A couple missing semicolons? May need to check linter config (or run it if you'd forgotten).

File: cqlDB/seed.js (hunk 2 above)
Comment (danzerca) [reply cqlDB/seed.js:89]: good catch. i was thinking i might want access to some of these functions later on when using new dbs. i will add them as keys instead. i might remove them later if I don't need them for setting up Cassandra and postgres.

File: cqlDB/seed.js (hunk 1 above)
Comment (danzerca) [reply cqlDB/seed.js:6]: that makes sense. i would only need it in my load script, not the dataGen script. i will take it out.

**Pull request merged**